streams-contrib/streams-persist-elasticsearch/src/main/java/org/apache/streams/elasticsearch/ElasticsearchQuery.java

```diff
@@ -99,6 +94,10 @@ public void setBatchSize(int batchSize) {
     }
 
     public void setScrollTimeout(String scrollTimeout) {
+        this.scrollTimeout = TimeValue.parseTimeValue(scrollTimeout, TimeValue.timeValueMillis(5 * 1000 * 60));
```

Comment:
Yes, there is a default in the schema. Default in that call too.. The second parameter is if the default fails.